tests/integration/utils/utils.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+
+import cunumeric as num
+
+
+def compare_array(b, c):
+    """
+    Compare two array using zip method.
+    """
+
+    is_equal = True
+    err_arr = [b, c]
+
+    if len(b) != len(c):
+        is_equal = False
+        err_arr = [b, c]
+    else:
+        for each in zip(b, c):
+            if not np.array_equal(*each):
+                err_arr = each
+                is_equal = False
+                break
+    return is_equal, err_arr
```

Comment:
It think this could be simplified:
```suggestion
    if len(b) != len(c):
        return False, [b, c]

    for each in zip(b, c):
        if not np.array_equal(*each):
            return False, each

    return True, None
```

Reply:
I'm also just curious, is the point of this comparison to get better detail about which top-level sub-arrays are not equal?

Reply:
I think so. I'm wondering why the function is called in many integration tests.

Reply:
I expect it is in order to obtain better diagnostic output in cases where there is a failure